### PR TITLE
Fixed an issue with langchange when SAFE_REDIRECT_HOSTS is missing

### PR DESCRIPTION
**Pull request by OmarIthawi**
Reopening #124 on master.

---

A perfectionist fix if the config is missing.
- Task: [Beta Environment: "Error 500" page is displayed when change the site language](https://app.asana.com/0/13296136706033/63272639987963)

### CODE REVIEW

**Comment by ahmedaljazzar** [comment]
@OmarIthawi, Is this still wanted, because I couldn't reproduce it locally and on [Edraakbeta](www.edraakbeta.org)? I also see that it's marked completed on Asana.

**Comment by OmarIthawi** [comment]
Sure it is. However, it only breaks after a configuration issue both on
devstack and production.

File: common/djangoapps/edraak_i18n/helpers.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from django.utils.http import is_safe_url
 
 
-def get_any_safe_url(hosts, urls):
+def get_any_safe_url(hosts, urls, fallback):
```
Comment (ahmedaljazzar) [inline common/djangoapps/edraak_i18n/helpers.py]: Nice it's only a fallback, why don't we give the argument a default value like `fallback ='/'`? This helps us reducing the code change later.

**Comment by OmarIthawi** [comment]
After a second thought. Let's close this PR. Not so sure anymore if it is needed anymore. We'll probably need a similar solution in [eucalyptus](github.com/Edraak/edraak-platform), possibly using edX's provided APIs.

**Comment by ahmedaljazzar** [comment]
lol 😂😂😂😂😂

**Pull request closed without merging**